test/testdata/bootstrappolicy/bootstrap_cluster_roles.yaml

```diff
@@ -62,6 +62,7 @@ items:
     - persistentvolumes/status
```

Comment:
Can you squash this commit along with 'Add scheduled jobs to all roles, new endpoints to cluster-reader role' ?